package.json

```diff
@@ -10,34 +10,32 @@
     },
     "scripts": {
         "analyze": "lit-analyzer \"{packages,tools}/*/src/**/!(*.css).ts\"",
+        "analyze:docs": "cem analyze --globs \"packages/**/*.ts\" --exclude \"**/*.d.ts\" --exclude \"**/stories/**\" --exclude \"**/icons/**\" --exclude \"**/elements/**\" --outdir projects/documentation --litelement",
```

Comment:
This was previous `docs: analyze` but I was aiming to group the cem tasks together so we can get a view of the manifests being generated.